src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -22,10 +22,16 @@
         <dependency id="System.ObjectModel" version="$SystemObjectModelVersion$" />
         <dependency id="System.Text.RegularExpressions" version="$SystemTextRegularExpressionsVersion$" />
         <dependency id="System.Threading.Tasks.Parallel" version="$SystemThreadingTasksParallelVersion$" />
+        <dependency id="SQLitePCLRaw.bundle_green" version="$SQLitePCLRawbundle_greenVersion$" />
+        <dependency id="Microsoft.Build" version="$MicrosoftBuildVersion$" />
```

Comment:
This should be gone now due to @DustinCampbell's change.

Reply:
Search in the project, it is not used. Removed from both the nuspec and the csproj

Reply:
@jasonmalinowski @DustinCampbell Actually Microsoft.Build is still used in the project, see Workspaces\Core\Desktop\Workspace\MSBuild\CSharp\CSharpProjectFileLoader.CSharpProjectFile.cs

Reply:
Yes, my changes are only in master. This should go away when this merges to master.

Reply:
Since we've changed to master, will need this change too now.